test/test/parser/Cpu.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.parser;
+
+import java.io.File;
+
+public class Cpu {
+    private static volatile int value;
+
+    private static void method1() throws Exception {
+        long startTime = System.currentTimeMillis();
+        while (System.currentTimeMillis() - startTime < 10) {
+            for (String s : new File("/tmp").list()) {
```

Comment:
`/tmp` might be emtpy.

Reply:
Removed in relation to Andrei's comment about not accessing file system